src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -590,13 +539,13 @@ ColorScheme* CascadiaSettings::_FindMatchingColorScheme(const Json::Value& schem
 {
     for (auto& scheme : _globals.GetColorSchemes())

```

Comment:
nit: file a followup here. now that we have a map, no need to iterate all of them. key=name=able to find layering scheme